### PR TITLE
Add MetalLB info in the release notes

### PR DESCRIPTION
**Pull request by camille-rodriguez**
- Add MetalLB Operator info
- Add link to upstream docs
- Add link to charm github
Do we need to add a link to the charmstore once the charm will be published?

### CODE REVIEW

File: pages/k8s/1.19/release-notes.md [hunk 1]
```diff
@@ -76,6 +76,15 @@ the new [SR-IOV CNI documentation][cni-sriov].
 For a full list of the changes introduced in Kubernetes 1.19, please see the
 [upstream release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md)
 
+- MetalLB Operator
```
Comment (tvansteenburgh) [inline pages/k8s/1.19/release-notes.md:79]: Let's move this up, right after the SR-IOV paragraph ("For a full list of the changes..." should come last).

File: pages/k8s/1.19/release-notes.md [hunk 2]
```diff
@@ -76,6 +76,15 @@ the new [SR-IOV CNI documentation][cni-sriov].
...
+
+MetalLB offers a software network load balancing implementation that allows for 
+LoadBalancing services in Kubernetes. This charm bundle has been made available
```
Comment (tvansteenburgh) [inline pages/k8s/1.19/release-notes.md:82]: s/charm//

File: pages/k8s/1.19/release-notes.md [hunk 3]
```diff
@@ -76,6 +76,15 @@ the new [SR-IOV CNI documentation][cni-sriov].
...
+in the charm store to be deployed along Charmed Kubernetes or MicroK8s. This operator
```
Comment (tvansteenburgh) [inline pages/k8s/1.19/release-notes.md:83]: ...to be deployed on Charmed Kubernetes, MicroK8s, or any Kubernetes supported by Juju.

File: pages/k8s/1.19/release-notes.md [hunk 4]
```diff
@@ -76,6 +76,15 @@ the new [SR-IOV CNI documentation][cni-sriov].
...
+deploys upstream MetalLB in layer 2 mode. The BGP mode of upstream MetalLB is not
+supported yet. For more information about deploying and operating MetalLB, please
+see the operator [README](https://github.com/charmed-kubernetes/metallb-operator) and the [upstream documentation](https://metallb.universe.tf/).
```
Comment (tvansteenburgh) [inline pages/k8s/1.19/release-notes.md:86]: We should link to https://ubuntu.com/kubernetes/docs/metallb, and let *that* page include links to the readme and upstream docs.

**Review by tvansteenburgh — CHANGES_REQUESTED** [review]
Thanks!

**Review by evilnick — APPROVED** [review]
:cake: 

I will add this to the u.c PR too

**Pull request merged**